recipes-openxt/openxt-measuredlaunch/openxt-measuredlaunch/seal-system

```diff
@@ -1,20 +1,33 @@
 #!/bin/bash
 
-. /usr/lib/openxt/ml-functions
-[ $? -eq 0 ] || {
-    echo "failed to load ml-functions"
+# Usage: err MESSAGE
+# Print MESSAGE on stderr and end the script with an error code 1.
+err() {
+    echo "seal-system: $1" >&2
     exit 1
 }
 
+# Load openxt-measuredlaunch helpers.
+if [ ! -f /usr/lib/openxt/ml-functions ]; then
+    err "Failed to load ml-functions. \
+        Is openxt-measuredlaunch installed correctly?"
+fi
+. /usr/lib/openxt/ml-functions
+
+# Load xenclient-tpm-scripts helpers.
+if [ ! -f /usr/lib/tpm-scripts/tpm-functions ]; then
+    err "Failed to load tpm-functions. \
+        Is xenclient-tpm-scripts installed correctly?"
+fi
+. /usr/lib/tpm-scripts/tpm-functions
```

Comment:
ml-functions already sources `/usr/lib/tpm-scripts/tpm-functions` & errors with a plain echo & exit 1: https://github.com/OpenXT/xenclient-oe/blob/master/recipes-openxt/openxt-measuredlaunch/openxt-measuredlaunch/ml-functions#L26

Reply:
Indeed, which drops the `RDEPENDS` on `xenclient-tpm-scripts` in the mean time.